listenstore/listenstore/listenstore.py

```diff
@@ -72,12 +71,12 @@ def convert_row(self, row):
                       recording_msid=row[5], data=row[6])
 
     def format_dict(self, listen):
-        return { 'user_id': listen.user_id,
-                 'ts': listen.timestamp,
-                 'artist_msid': uuid.UUID(listen.artist_msid),
-                 'album_msid': uuid.UUID(listen.album_msid) if listen.album_msid is not None else None,
-                 'recording_msid': uuid.UUID(listen.recording_msid),
-                 'raw_data': ujson.dumps(listen.data)}
+        return {'user_id': listen.user_id,
+                'ts': listen.timestamp,
+                'artist_msid': uuid.UUID(listen.artist_msid),
+                'album_msid': uuid.UUID(listen.album_msid) if listen.album_msid is not None else None,
+                'recording_msid': uuid.UUID(listen.recording_msid),
+                'data': ujson.dumps(listen.data)}
 
     def insert_postgresql(self, listens):
```

Comment:
`insert_postgresql` can be just called `insert` or `add`. The class is `PostgresListenStore` so everything inside uses PostgreSQL by default, there's no need to specify it here.